### ISSUE TITLE
Devstack integration doesn't add /opt/stack/go/bin to $PATH

### ISSUE THREAD

**Issue by assafmuller** [skydive-project/skydive#1500]
Just finished a stack.sh on a fresh Fedora 29 VM with the following local.conf:

```
[[local|localrc]]

FORCE=yes
enable_plugin skydive https://github.com/skydive-project/skydive.git
enable_service skydive-agent skydive-analyzer

ADMIN_PASSWORD=1
DATABASE_PASSWORD=1
RABBIT_PASSWORD=1
SERVICE_PASSWORD=$ADMIN_PASSWORD

LOGFILE=$DEST/logs/stack.sh.log
LOGDAYS=2

SWIFT_HASH=66a3d6b56c1f479c8b4e70ab5c2000f5
SWIFT_REPLICAS=1
SWIFT_DATA_DIR=$DEST/data
```

It finishes successfully, the skydive processes are up, however the skydive client is no where to be found. My $PATH didn't seem to include /opt/go/*, and /opt/go/bin/ did not include a skydive binary.

**Comment by lebauce** [comment]
@assafmuller The skydive binary should be in /opt/stack/go/bin
/opt/go/bin is for the go compiler itself

**Comment by assafmuller** [comment]
My bad :) The only issue then is that the skydive devstack plugin doesn't amend /opt/stack/go/bin to $PATH.

**Comment by safchain** [comment]
fixed: #1506
